projects/infrastructure-for-perf-tracking.adoc

```diff
@@ -101,9 +110,10 @@ Contributors taking this project needs:
 === Deliverables
 
 * An open-source toolset for building compilers & running benchmarks & show perf results on web.
-* Data reports generated by this tracking platform.
+* Potential optimization opportunites and regressions uncovered by the PTS.
+* Annual progrss report of all the compilers, runtimes, libraries under the tracking of the PTS.
 
-A monthly report to the SIG is a routine deliverables for all projects, which will cover
+A (bi-)monthly report to the SIG is a routine deliverables for all projects, which will cover
```

Comment:
Doesn't need a change, but since the SIG meetings monthly, one of these will be an email report, the other will be an email report, with a verbal summary at the meeting.